addon/-private/modifier.ts

```diff
@@ -564,28 +568,26 @@ export default class ElementQueryModifier extends Modifier<ModifierArgs> {
     debounceTask(this, '_didResizeHandlerRequestAnimationFrame', this.debounce);
   }
 
-  // -------------------
-  // Lifecycle hooks
-  // -------------------
-  didInstall(): void {
-    if (!this.element) throw new Error('Expected this.element to be available');
-
-    this._element = this.element;
-
-    this.resizeObserver.observe(this.element, this.didResizeHandler); // eslint-disable-line @typescript-eslint/unbound-method
-
-    this._didResizeHandlerSync(); // We want to run it as soon as possible to avoid a jump of unstyled content
-  }
-
-  didUpdateArguments(): void {
-    if (!this.resizeObserver.isEnabled) return;
+  modify(element: HTMLElement, _pos: unknown[], named: Args): void {
+    this._element = element;
+    this.named = named;
+    if (this._firstCall) {
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      this.resizeObserver.observe(element, this.didResizeHandler);
+      this._didResizeHandlerSync();
 
-    this.didResizeHandler();
+      registerDestructor(this, () => this.cleanup());
```

Comment:
Why this change (arrow function)? In order to remove the lint exception? But we have this exact lint exception used all over the file, so I don't see a point in fixing just one instance.

The linter should serve to the developer, not vice versa. :)